docs/concepts/alerts.md

```diff
@@ -0,0 +1,128 @@
+(alerts)=
+# Alerts 
```

Comment:
we should add in this section a description for each of the alert config fields.

Reply:
added link to SDK